test/unit/lib/plugins/aws/invoke-local/index.test.js

```diff
@@ -997,7 +998,14 @@ describe('test/unit/lib/plugins/aws/invokeLocal/index.test.js', () => {
             },
           },
         });
-        const outputAsJson = JSON.parse(response.output);
+        const outputAsJson = (() => {
```

Comment:
Is there a reason why this change landed here?

Reply:
Yes, sorry for not providing an explanation. There was a CI test fail, where this parse failed due to not uniform JSON. This log was added to give a more clue, next time it happens

https://github.com/serverless/serverless/runs/5583504964?check_suite_focus=true

Reply:
Thanks for clarification 🙇